admin/src/js/services/search.js

```diff
@@ -1,5 +1,4 @@
-const _ = require('lodash/core');
-const moment = require('moment');
+const _ = require('lodash'); // #8494 don't use eslint/core as it throws an exception
```

Comment:
This was the fix. The bug only impacts the "core" bundle. The downside is that the bundle gets bigger, but not by much.